### ISSUE TITLE
Idea on Cum Addict fetish

### ISSUE THREAD

**Issue by ThatWhichLurks** [Innoxia/liliths-throne-public#813]
It's nothing big, but it's been bugging me for a while: Why is the fetish called "Cum ADDICT" even though it doesn't really act like an addiction? Could it maybe be reworked, so that the fetish for receiving cum-play is called “Cum Slut,” and then "Cum Addict" is an enhanced version (taken as a trait, probably; for balance reasons it shouldn’t be something one character can force on another) that gives an incurable true addiction to sexual fluids (any cum or girlcum except your own sates it, Angel's Nectar has no effect) and significantly larger buffs while under any creampie status (including Cummy Meal but not Yummy Meal)?

Here's my rough sketch:

Cum Slut Fetish
"You can't get enough of being covered and filled with sexual fluids. The taste, the smell, the feeling... you love it all."
-Replaces dirty and creampied status effects with beneficial versions
-Unlocks cum slut tease, weak to cum stud tease

Cum Addict Trait
Prerequisite: have the “Cum Slut” fetish
"You are hopelessly addicted to others' sexual fluids.  Guzzling cum straight from the source is your greatest thrill and you always crave more.  After one binge too many, you've also become quite tolerant of being overstuffed."
-Greatly enhances creampie status effects
-Extreme, incurable cum addiction
-Increased inflation penalty thresholds (inflation thresholds become 250% normal)

Cum High status effect
Granted by Cum Addict trait while under any creampie effect
"Being filled with cum gives you a rush like no other... but you'd still like to have even more"
+5 Phys, + 2 Arc, + 10 Melee Damage, +10 Ranged Damage, + 20 Lust Damage

Cum Junkie status effect
Inflicted by Cum Addict trait when a creampie effect ends, unless still under another creampie effect
"Having no more cum in you makes you sad.  It won't be long before you can't think of anything but getting more...."
After 3 hours, replaced by "Cum Craving"

Cum Craving status effect
Inflicted when Cum Junkie wears off
"You can't take it any more! You need cum!  You're starting to have trouble focusing on anything else."
-1 Phys, -1 Arc, weak to virility, cum stud, and oral teases, minimum lust 50, +50% lust damage dealt and taken.  After 9 hours, also inflicts Cum Withdrawal (minor).

[severity] Cum Withdrawal status effects
(As normal for [severity] Withdrawal, except progressing twice as fast.)

For NPCs: all NPCs generates with the Cum Slut fetish have a 10% chance to also have the new trait.  At each encounter they have a 50% chance of being in withdrawal.  I’m going to post a separate Issue for NPC addict behavior ideas.

**Comment by CognitiveMist** [comment]
I've always thought it should be "Cum Slut", which nicely mirrors "Cum Stud".

**Comment by S-abLE** [comment]
Quality Idea.

**Comment by ThatWhichLurks** [comment]
@CognitiveMist: even better, editing to your suggestion.
